proposals/xxxx-continous-integration.md

```diff
@@ -0,0 +1,73 @@
+# Configuration management
+
+* Proposal: [SDL-NNNN](xxxx-Configuration-Management.md)
+* Author: [Vadym Kushnirenko](https://github.com/vkushnirenko-luxoft)
+* Status: **Awaiting review**
+* Impacted Platforms: [Build server]
+
+## Introduction
+
+This proposal is about an order of creating build jobs in our buiding system, running tests and keeping results of theese tests.
+Also a good practice to provide a set of scripts to easy manage our build system.
+
+## Motivation
+
+The building process should be transparent. Any person should be able to deploy build server on his side, use it and propose recomendations about improvements.
+Build server configuration must be saved into files, be flexible and be applied in any moment from public resources.
+
+## Proposed solution
+
+Key components of build system are:
+- jenkins master - it manages build/test processes,
+- jenkins slaves - servers which builds and tests,
+- artifactory - keeps archive with binaries and log files.
+- Github - keeps Dockerfiles and Jenkins jobs, allows to keep different versions of configuration and to download a configuration at any time.
+The solution proposes to use docker containers for components deployment. 
+### Jenkins jobs creating.
+Every task that implies alterations in Git repository should be represented as separated branch. The template for naming the branch is the following:
+feature/<task name> , fix/<task name> or hotfix/<task name>
```

Comment:
@vkushnirenko-luxoft Seems ```<task name>``` was removed during formatting

Reply:
Rejected. What do you mean?

Reply:
@vkushnirenko-luxoft When .md file is in view mode the following is displayed:
" The template for naming the branch is the following: feature/ , fix/ or hotfix/ "
```<task name>``` is removed during formatting